### ISSUE TITLE
Modify a process instance by activating root elements

### ISSUE THREAD

**Issue by saig0** [camunda/zeebe#9640]
## Description

We want to support the basic modification by activating root elements (i.e. elements that are not embedded in a subprocess). We want to create a new element instance for each activate instruction. The element instances should be created by writing an `activate` command for the given elements. As a result, the process instance should continue at the given elements.

For example:
![image](https://user-images.githubusercontent.com/4305769/176377455-df415e0e-48cd-45ba-8e9c-5bb7571a0f6c.png)

* write an `activate` command for each activate instruction
* verify that 
  * the process instance activates the given elements
  * the process instance can complete the given elements
  * the process instance is completed when the given elements and the other element instances are completed

Out of scope:
* nested elements inside subprocesses
* not supported elements (see #9729)

Blocked by #9639
